file_sizes.txt

```diff
@@ -0,0 +1,22 @@
+
+List of File Sizes
+==================
+This is to track progress on trimming code size. Goal of DIS-28 is to slim down overall size of iframe and associated code.
+
+
+2/12/2019
+
+digitrust.min.js		38.7kb
+dt.html					5.25kb
+dt-embed.html			38.2kb
+
+
+4/3/2019
+
+digitrust.min.js			36.9kb
+digitrust_iframe.min.js		37.6kb
```

Comment:
what's the point of a separate digitrust_iframe.min.js if the size is even larger than digitrust.min.js!?

Reply:
This is a partial port of the codetrim branch. Since the master branch has diverged so much I'm going to abandon that branch and bring things over piecemeal. This file is to track progress and not lose it in some corner of some ticket.

The "digitrust_iframe.min.js" file is a work in progress tied to [DIS-28](https://theiab.atlassian.net/browse/DIS-28). The end goal is to directly embed only the script needed to run the iframe instead of bringing in the entire library. This will save us a network request (theoretically). I was tracking the file sizes as code was being refactored to make sure we were moving in a positive direction. After unwinding the dependencies and splitting what is front-end (pub side) from what is iframe (DT domain) and what is common (if anything), I want us to be able to make an informed decision.

Reply:
I wonder if the request for digitrust.min.js from within dt.html hits browser cache... if it does, aren't we better off with a second, duplicate request, rather than embedding 30k (or whatever) of code within dt.html? Is there a data-driven method to figure that out?